README.md

```diff
@@ -1 +1,113 @@
-# math-magician
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math Magician
+
+## Built With
+
+- HTML5
+- CSS
+- JavaScript
+- React
+
+
+### Setup
+
+- Clone this repo URL "https://github.com/vanillainanutshell/math-magician"
+
+## Authors
```

Comment:
- [ ] OPTIONAL
To improve the professionalism aspect of your ReadMe, kindly change the header title _Authors_ to _Author_, as this project is done solely.